packages/env/README.md

```diff
@@ -68,3 +68,28 @@ Positionals:
 ```
 
 <br/><br/><p align="center"><img src="https://s.w.org/style/images/codeispoetry.png?1" alt="Code is Poetry." /></p>
+
+## The Configuration File
+
+You may also specify a configuration file for `wp-env`. This can be useful for loading other themes and plugins that you are developing simultaneously. The script will attach the specified theme and plugin directories as volumes on the docker container so that changes you make to them exist in the WordPress instance.
```

Comment:
```suggestion
`wp-env` also supports a configuration file. At the moment, this is only used for loading extra themes and plugins that you may be developing together with your main one. The script will attach the specified theme and plugin directories as volumes on the docker containers so that changes you make to them exist in the WordPress instance.
```